src/routes/api/user/+server.ts

```diff
@@ -1,32 +1,35 @@
 import { authenticate, getOrganizationInfo, getUserData, getUserOrganizations } from '$lib/server/firebaseUtils';
-import {
-  getBlendProSubscription,
-  getStripeCustomerWithSubscriptions,
-  isOrganizationMember,
-  isSubscribedToBlendPro,
-} from '$lib/server/subscriptionUtils';
+import { isSubscribedToBlendPro } from '$lib/server/subscriptionUtils';
 import type { RequestHandler } from './$types';
 
 export const GET = (async (event) => {
+  const startTime = Date.now();
   const { uid } = await authenticate(event);
-  const [stripeCustomer, firebaseUserData] = await Promise.all([getStripeCustomerWithSubscriptions(uid), getUserData(uid)]);
-  const subscriptionData = stripeCustomer && !stripeCustomer.deleted && getBlendProSubscription(stripeCustomer);
-  const userOrganizations = await getUserOrganizations(uid);
-  const organizationInfo = await Promise.all(
-    userOrganizations.map(async (orgId) => {
-      const orgInfo = await getOrganizationInfo(orgId);
-      return {
-        orgName: orgInfo?.name,
-        orgId,
-      };
-    }),
-  );
+  const authenticateTime = Date.now();
+  console.log(`Authenticated in ${authenticateTime - startTime}ms`);
+  const [firebaseUserData, isProSubscriber, organizationInfo] = await Promise.all([
+    getUserData(uid),
+    isSubscribedToBlendPro(uid),
+    getUserOrganizations(uid).then((orgs) =>
+      Promise.all(
+        orgs.map(async (orgId) => {
+          const orgInfo = await getOrganizationInfo(orgId);
+          return {
+            orgName: orgInfo?.name,
+            orgId,
+          };
+        }),
+      ),
+    ),
+  ]);
```

Comment:
Without Stripe in the way, all of these can be inlined. I did the extra step of fetching the org info in a `.then` because we may be waiting on one of the others, so may as well get started on that.